### PR TITLE
[SERVICE-865] Move "keep centre open"/"do not disturb" functionality into settings UI

### PR DESCRIPTION
**Pull request by ryanpetersonOF**
As title says, moves the auto hide and do not disturb options into the new settings UI

### CODE REVIEW

File: src/provider/view/components/SettingsView/SettingsView.tsx [hunk 1]
```diff
@@ -15,17 +17,23 @@ const mockFeed: Feed = {
 const mockFeeds = new Array(5).fill(mockFeed);
 
 export const SettingsView: React.FC = (props) => {
+    const storeApi = usePreduxStore();
```
Comment (TheoOttahOF) [inline src/provider/view/components/SettingsView/SettingsView.tsx]: SettingsPanel/SettingsView should be now connected to the store and map the state (locked/muted) into the component to update properly.

**Comment by ryanpetersonOF** [comment]
@bryangaleOF Yes I see those tests 😫

File: src/provider/view/components/SettingsView/SettingsView.tsx (hunk 1 above)
Comment (ryanpetersonOF) [reply src/provider/view/components/SettingsView/SettingsView.tsx]: ✅

**Comment by ryanpetersonOF** [comment]
@bryangaleOF Tests are done, ready for re-review

File: test/utils/int/centerUtils.ts [hunk 2]
```diff
@@ -49,10 +52,14 @@ async function toggleCenterSetting(elementId: string): Promise<void> {
         await delay(Duration.CENTER_TOGGLED);
     }
 
+    await navigateCenter(ROUTES.SETTINGS);
```
Comment (bryangaleOF) [inline test/utils/int/centerUtils.ts]: This is really neat!

**Pull request merged**